messagebus/src/main/java/com/yahoo/messagebus/MessageBus.java

```diff
@@ -511,14 +511,6 @@ public void setMaxPendingSize(int maxSize) {
         maxPendingSize = maxSize;
     }
 
-    /**
-     * Gets maximum combined size of messages that can be received without
-     * being replied to yet.
-     */
-    public int getMaxPendingSize() {
-        return maxPendingSize;
-    }
-
```

Comment:
Can we remove these or change their accessibility when they're public functions in a `@PublicApi`/`@ExportPackage` package?